haystack/nodes/extractor/entity.py

```diff
@@ -34,10 +39,11 @@ def __init__(
         use_gpu: bool = True,
         batch_size: int = 16,
         progress_bar: bool = True,
+        devices: Optional[List[Union[str, torch.device]]] = None,
     ):
         super().__init__()
 
-        self.devices, _ = initialize_device_settings(use_cuda=use_gpu, multi_gpu=False)
+        self.devices, _ = initialize_device_settings(devices=devices, use_cuda=use_gpu, multi_gpu=False)
```

Comment:
Also, consider adding an assert here for devices to be length 1 since we only support a single device for the model to be loaded onto. 

Additionally below we have the line `device=0 if self.devices[0].type == "cuda" else -1` which makes me concerned that `mps` would not be supported for loading this model.

Reply:
Yes, not supported on mps right now. Will make the notes for the components where only cpu and cuda are supported.